### PR TITLE
Groups page

### PR DESCRIPTION
**Pull request by Vagabonde**
Added pop-up component and functionality to add new group. Could be updated by adding a service.

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ import {GroupComponent} from './group/group.component';
 import {HomepageComponent} from './homepage/homepage.component';
 import {ModalLoginComponent} from './modal-login/modal-login.component';
 import {ModalSignUpComponent} from './modal-sign-up/modal-sign-up.component';
+import {ModalAddGroupComponent} from "./groups/modal-add-group/modal-add-group.component";
```
Comment (reshetniktati) [inline src/app/app.module.ts]: please use single quotes

File: src/app/groups/groups.component.html [hunk 2]
```diff
@@ -1,28 +1,31 @@
...
         <h1 class="mb-5 font-weight-bold ml-2">Select group</h1>
         <div class="row align-content-space-evenly justify-content-around justify-content-sm-start m-0">
             <div class="col-12 col-md-4 col-lg-3 mb-4 p-2" *ngFor="let group of groups">
                 <div class="card">
-                    <div class="img-wrapper p-2">
-                        <img class="card-img-top w-100" src="../../assets/{{group.img}}" alt="Group image">
-                    </div>
+                    <a [routerLink]="['/groups', group.id]">
+                        <div class="img-wrapper p-2">
+                            <img class="card-img-top w-100" src="../../assets/{{group.img}}" alt="Group image">
```
Comment (reshetniktati) [inline src/app/groups/groups.component.html]: Would it be better to use absolute Url (`src="/assets/{{group.img}}"`)?

File: src/app/groups/groups.component.scss [hunk 3]
```diff
@@ -49,26 +65,28 @@ h2 {
...
-.btn {
-  background-color: transparent;
-  box-shadow: none;
-  //box-sizing: border-box;
+.btn-add {
 
   .btn-img {
-    background: #fff url("../../assets/icon-plus.png") no-repeat;
-    background-size: cover;
+    background: #fff url("../../assets/icon-plus.png") no-repeat center center;
```
Comment (reshetniktati) [inline src/app/groups/groups.component.scss]: absolute URL?

File: src/app/groups/groups.component.scss [hunk 4]
```diff
@@ -49,26 +65,28 @@ h2 {
...
 }
 
-.btn:hover {
+.btn-add:hover {
   border-color: $hoverColor;
+
   .btn-img {
-    background: #fff url("../../assets/icon-plus-active.png") no-repeat;
-    background-size: cover;
+    background: #fff url("../../assets/icon-plus-active.png") no-repeat center;
```
Comment (reshetniktati) [inline src/app/groups/groups.component.scss]: absolute Url?

File: src/app/groups/modal-add-group/modal-add-group.component.ts [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import {Component, OnInit} from '@angular/core';
+import {GROUPS} from "../../mock-groups";
```
Comment (reshetniktati) [inline src/app/groups/modal-add-group/modal-add-group.component.ts]: single quotes?

File: src/app/groups/groups-page.module.ts [hunk 6]
```diff
@@ -1,23 +1,26 @@
 import {CommonModule} from '@angular/common';
 import {GroupsComponent} from "./groups.component";
 import {RouterModule} from "@angular/router";
-import {NgModule, NO_ERRORS_SCHEMA} from '@angular/core';
```
[the remark this answers is not in the document]
Comment (Vagabonde) [reply src/app/groups/groups-page.module.ts]: According to official documentation it only needs to be imported in app module. Please, check https://github.com/mdbootstrap/Angular-Bootstrap-with-Material-Design and correct me if i am wrong.

[the remark this answers is not in the document]
Comment (reshetniktati) [reply src/app/groups/groups-page.module.ts]: ok, I see.
thank you for the explanation)

File: src/app/app.module.ts (hunk 1 above)
Comment (Vagabonde) [reply src/app/app.module.ts]: Fixed

File: src/app/groups/groups.component.html (hunk 2 above)
Comment (Vagabonde) [reply src/app/groups/groups.component.html]: You're right! Fixed

File: src/app/groups/groups.component.scss (hunk 3 above)
Comment (Vagabonde) [reply src/app/groups/groups.component.scss]: Fixed

File: src/app/groups/groups.component.scss (hunk 4 above)
Comment (Vagabonde) [reply src/app/groups/groups.component.scss]: Fixed

File: src/app/groups/modal-add-group/modal-add-group.component.ts (hunk 5 above)
Comment (Vagabonde) [reply src/app/groups/modal-add-group/modal-add-group.component.ts]: Fixed

**Pull request merged**